EIPS/eip-6224.md

```diff
@@ -4,7 +4,7 @@ title: Contracts Dependencies Registry
 description: An interface for managing smart contracts with their dependencies.
 author: Artem Chystiakov (@arvolear)
 discussions-to: https://ethereum-magicians.org/t/eip-6224-contracts-dependencies-registry/12316
-status: Draft
+status: Review
 type: Standards Track
 category: ERC
 created: 2022-12-27
```

Comment:
This comment goes to next line: 
```
requires: 1967, 5750
```
Maybe I am missing something here. 

It seems I couldn't find the mentioning of 1967 in this EIP. This EIP also seemed lack mentioning the requirement of a proxy. If a proxy is a ERC-1967 proxy, it doesn't need any specific function to retrieve the impl address because you can get it from ERC-1967.